lms/models/application_settings.py

```diff
@@ -25,7 +24,13 @@ def set(self, group, key, value):
         :param key: The key in that group
         :param value: The value to set
         """
-        self.data.setdefault(group, {})[key] = value
+        super().setdefault(group, {})[key] = value
 
     def clone(self):
-        return ApplicationSettings(deepcopy(self.data))
+        return ApplicationSettings(deepcopy(self))
+
+    def __repr__(self):
+        return f"ApplicationSettings({super().__repr__()})"
+
+    def __str__(self):
+        return repr(self)
```

Comment:
Added these to make `ApplicationSettings` objects easily distinguishable from normal dicts